### PR TITLE
feat: add klarna provider

### PR DESCRIPTION
**Pull request by taher435**
Add Klarna login provider

### CODE REVIEW

File: src/Klarna/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## Installation & Basic Usage
+
+Please see the [Base Installation Guide](https://socialiteproviders.com/usage/), then follow the provider specific instructions below.
+
+### Add configuration to `config/services.php`
+
+```php
+'klarna' => [
+  'client_id' => env('CLIENT_ID'),
```
Comment (lucasmichot) [inline src/Klarna/README.md:15]:
```suggestion
  'client_id' => env('KLARNA_CLIENT_ID'),
```

File: src/Klarna/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  'client_secret' => env('CLIENT_SECRET'),
```
Comment (lucasmichot) [inline src/Klarna/README.md:16]:
```suggestion
  'client_secret' => env('KLARNA_CLIENT_SECRET'),
```

**Review by lucasmichot — CHANGES_REQUESTED** [review]
Thanks for the PR, can you also add an entry to https://github.com/SocialiteProviders/Providers/blob/master/monorepo-builder.yml ?

**Review by lucasmichot — APPROVED** [review]
GTM @atymic

**Comment by lucasmichot** [comment]
@taher435 , can you make also a PR on https://github.com/SocialiteProviders/website ?

**Comment by taher435** [comment]
> @taher435 , can you make also a PR on https://github.com/SocialiteProviders/website ?

@lucasmichot done. https://github.com/SocialiteProviders/website/pull/69

**Comment by taher435** [comment]
> GTM @atymic

Hey @atymic are you able to review this PR? Thank you 🙏

**Review by lucasmichot — APPROVED** [review]
Reviewed again after

**Pull request merged**